kod/object/passive/itematt/weapatt/wavamper.kod

```diff
@@ -58,31 +57,37 @@ messages:
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
       
    ModifyDamage(damage = 0, target = $, wielder = $, lData= $)
-   "Wielder has a 1% chance of stealing life from target with any hit."
+   "Wielder has a 10% chance of stealing life from target with any hit."
```

Comment:
If somebody else changes piEffect_percent from 10 to something else sometime in the future, this comment becomes incorrect again, just like it was by stating a value of 1%.  It might be better to state that the chance is based on piEffect_percent.

Reply:
Good thinking ok I made the change